src/point.rs

```diff
@@ -185,6 +186,30 @@ impl Point {
     pub fn is_nan(self) -> bool {
         self.x.is_nan() || self.y.is_nan()
     }
+
+    /// The topmost [`Point`]
+    #[inline]
+    pub fn topmost(&self, other: &Self) -> Ordering {
```

Comment:
yes i think this was pointed out by @raphlinus but i ended up keeping the names while `organise()` ([8b03193](https://github.com/linebender/kurbo/pull/350/commits/8b03193f6f932807b723c9d693a4010ba5e5076d)) was still in use. i just forgot to change them since `organise()` was removed, sorry.